#!/usr/bin/env python

# set up parameters that we care about
PACKAGE = 'jsk_pcl_ros'

from dynamic_reconfigure.parameter_generator_catkin import *;

from math import pi

gen = ParameterGenerator ()

gen.add("search_radius", double_t, 0, "search radius", 0.01)

exit (gen.generate (PACKAGE, "jsk_pcl_ros", "PointCloudLocalization"))
